src/Collector.cs

```diff
@@ -127,19 +142,30 @@ private void SendQueuedMessages(Socket socket)
         }
     }
 
-    private void Authenticate(Socket socket)
+    private bool Authenticate(Socket socket)
     {
-      var data = System.Text.Encoding.ASCII.GetBytes("hello version 1.0\n");
+      var data = System.Text.Encoding.ASCII.GetBytes("hello version dotnet/0.2.0\n");
```

Comment:
Let's make the version into a constant of some sort, or load it from the package description if possible.